composer.json

```
@@ -18,19 +18,19 @@
         "aws/aws-sdk-php": "^3.80",
         "guzzlehttp/guzzle": "^6.3|^7.0",
         "hollodotme/fast-cgi-client": "^3.0",
-        "illuminate/container": "^6.0|^7.0|^8.0|^9.0",
-        "illuminate/http": "^6.0|^7.0|^8.0|^9.0",
-        "illuminate/queue": "^6.0|^7.0|^8.0|^9.0",
-        "illuminate/support": "^6.0|^7.0|^8.0|^9.0",
-        "monolog/monolog": "^1.12|^2.0",
+        "illuminate/container": "^6.0|^7.0|^8.0|^9.0|^10",
+        "illuminate/http": "^6.0|^7.0|^8.0|^9.0|^10",
+        "illuminate/queue": "^6.0|^7.0|^8.0|^9.0|^10",
+        "illuminate/support": "^6.0|^7.0|^8.0|^9.0|^10",
```

```suggestion
        "illuminate/container": "^6.0|^7.0|^8.0|^9.0|^10.0",
        "illuminate/http": "^6.0|^7.0|^8.0|^9.0|^10.0",
        "illuminate/queue": "^6.0|^7.0|^8.0|^9.0|^10.0",
        "illuminate/support": "^6.0|^7.0|^8.0|^9.0|^10.0",
```

Suggesting we use `10.0` rather than `10` to align with other packages such as [Horizon](https://github.com/laravel/horizon/blob/5.x/composer.json#L17)